src/lang/Messages_uk_UA.properties

```diff
@@ -1553,6 +1553,7 @@ report.write.overwrite.dialog.title=Existing File
 report.write.overwrite.dialog.message=The selected file already exists. Do you want to replace it?
 report.write.permission.dialog.title=Permissions Failure
 report.write.permission.dialog.message=File or Directory not writable\: \n{0}\n Please select a different location.
+report.write.diskspace.error.dialog.message=Not enough disk space to export file
```

Comment:
This should be added to Messages.properties file (which is the source of translations, like this file).
The message should probably say, save/write than export.